assets/scss/lib/partials/_search.scss

```diff
@@ -56,3 +56,56 @@
     display: none;
   }
 }
+
+// Search page form.
+#views-exposed-form-localgov-sitewide-search-sitewide-search-page {
+  max-width: 48em;
+  padding: 1em 2em 2em 2em;
+
+  .form-actions {
+    padding-left: 0;
+  }
+
+  @include media-breakpoint-up(md) {
+    .form-text {
+      float: left;
+      width: 80%;
+      height: 3em;
+      margin: 0;
+      padding: 0.5em 1em;
+    }
+
+    .form-actions {
+      clear: none;
+      float: right;
+      height: 3em;
+      width: 20%;
+      margin: 0;
+
+      .button {
+        margin-top: 0 !important;
+        padding-top: 0.45em;
+        padding-bottom: 0.4em;
+
+        &:after {
+          top: 0.5em;
+        }
+      }
+    }
+  }
+}
+
+// Search results.
+ol.search-results {
```

Comment:
We should not need to use the `ol` element here. Within BEM classes should be styled independent of what element they are placed on.

Adding default styling of an `element.class` means the base theme that wants to override this will have to use something that is _at least_ as specific as this, if not more specific.

Reply:
Thanks. This is now nested in the view class and have removed the `ol` element.